README.md

```diff
@@ -1,2 +1,51 @@
 # opengemini-client-java
-openGemini java client(sdk)
+
+opengemini-client-java is a Java client for OpenGemini. 
+
+English | [简体中文](README_CN.md)
+
+OpenGemini is an open-source time series database, find more about OpenGemini at https://github.com/openGemini/openGemini
+## Characteristic
+
+- This project is implemented independently by multiple components, which can be referenced separately according to different requirements.
+- This project is implemented in line with other SDKs such as opengemini-client-go and opengemini-client-python, adhering to the same API layer philosophy, resulting in a very similar user experience for different users.
+
+| Sdk Components | Runtime Minimum JDK Version| Recommended Scenario | Notice |
+| --------------------------------- | ------------------- | ------------------------------------ | ------------------------------------------------------------ |
+| opengemini-client-jdk             | jdk17               | Minimum Dependency Scenario          |The native httpclient of the jdk was introduced in jdk9, but as versions such as jdk9 and jdk11 are no longer supported, it is recommended to upgrade to the higher jdk17 version|
```

Comment:
OpenJdk11 is still supported, but  will end of life in October 2024.

Windows Server 2012 R2 Support Added | End of Full Support for OpenJDK version
-- | --
OpenJDK 8 (1.8) | August 2018 | November 2026
OpenJDK 11 | October 2018 | October 2024
OpenJDK 17 | November 2021 | October 2027
OpenJDK 21 | N/A | December 2029